src/test/java/it/mulders/mcs/search/WildcardSearchQueryTest.java

```diff
@@ -13,30 +13,17 @@
     @DisplayName("withLimit")
     class WithLimitTest {
         @Test
-        void null_limit_should_return_original_object() {
-            var query = new WildcardSearchQuery("foo");
+        void simple_query_should_return_wildcard_query() {
```

Comment:
I like this method name: very clear as to what you can expect the test to verify :-)